tasks/config/setup-logrotate.yml

```diff
@@ -11,10 +11,25 @@
   when: ansible_facts['os_family'] == 'Debian'
 
 - name: (Amazon Linux/CentOS/Oracle Linux/RHEL) Install logrotate
+  vars:
```

Comment:
I am not entirely sure we need this vars block? We can check whether it's Amazon Linux 2023 in the `when` block directly.